packages/nodes-base/nodes/Google/Gmail/v2/utils/draft.ts

```diff
@@ -4,27 +4,43 @@ import type { IEmail } from '@utils/sendAndWait/interfaces';
 
 import { googleApiRequest } from '../../GenericFunctions';
 
+function setThreadHeaders(
+	email: IEmail,
+	thread: { messages: Array<{ payload: { headers: Array<{ name: string; value: string }> } }> },
+): void {
+	if (thread?.messages) {
+		const lastMessage = thread.messages.length - 1;
+		const messageId = thread.messages[lastMessage].payload.headers.find(
+			(header: { name: string; value: string }) => header.name === 'Message-ID',
+		)?.value;
+
+		email.inReplyTo = messageId;
```

Comment:
The References and In-Reply-To headers must be set in compliance with the [RFC 2822](https://tools.ietf.org/html/rfc2822#appendix-A.2) standard.
```suggestion
	email.inReplyTo = `<${messageId}>`;
```